tests/test_operation.py

```diff
@@ -56,17 +58,19 @@ def test_change_debt(gov, token, vault, strategy, strategist, amount):
     vault.deposit(amount, {"from": gov})
     vault.updateStrategyDebtRatio(strategy.address, 5_000, {"from": gov})
     strategy.harvest()
+    half = int(amount / 2)
 
-    assert token.balanceOf(strategy.address) == amount / 2
+    assert pytest.approx(strategy.estimatedTotalAssets(), rel=1e-5) == half
```

Comment:
should the `rel` be half too?

Reply:
You have to way to use this `rel` or `abs` rel meaning relative it will "scale" with the number.

Reply:
ah okay